react/components/ProductKitContent.js

```diff
@@ -0,0 +1,94 @@
+import { equals } from 'ramda'
+import React, { Component, Fragment } from 'react'
+import { FormattedMessage } from 'react-intl'
+
+import { extractItemsKit } from '../helpers/ProductKitHelper'
+import ProductKitDetails from './ProductKitDetails'
+import ProductKitItem from './ProductKitItem'
+
+const MAX_VISIBLE_ITEMS = 3
+const DEFAULT_VISIBLE_ITEMS = Array(MAX_VISIBLE_ITEMS).fill(null)
+
+/**
+ * Product Kit component.
+ * Display a list of items which composes a kit.
+ */
+export default class ProductKitContent extends Component {
+  state = {
```

Comment:
Try to separate the components into containers with states and pure presentational components. Vide:

https://medium.com/@learnreact/container-components-c0e67432e005